client/src/component/CodeWarsCard.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export const CodeWarsCard = ({ rank, progress, milestone }) => {
+  return (
+    <div className="codewars-card">
+      <h2>CodeWars Milestone</h2>
+      <p>Your Rank: {rank}</p>
+      <p>You are on the track</p>
+      <p>Progress: {progress}%</p>
+      <p>You need to catch: {milestone}</p>
+    </div>
+  );
+};
+
+export { CodeWarsCard }
```

Comment:
delet this line. By using export const in line 3, you are exporting the CodeWarsCard component as a named export, which can be imported and used in other files using its specific name (CodeWarsCard in this case).

Reply:
when you edit your code at the end use Prettier to organise your code in this file: shift+option+f